### PR TITLE
fix(logging):Ensure all references of --project are now --project-file

### PR DESCRIPTION
**Pull request by Bigtalljosh**
As per issue #142

### CODE REVIEW

File: src/Stryker.CLI/Stryker.CLI/CLIOptions.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static class CLIOptions
 
         public static readonly CLIOption<string> ProjectName = new CLIOption<string>
         {
-            ArgumentName = "--project",
+            ArgumentName = "--project-file",
```
Comment (richardwerkman) [inline src/Stryker.CLI/Stryker.CLI/CLIOptions.cs]: Maybe it is confusing that the JsonKey is `project-name` and the argumentname is `--project-file`. I think they should be in line.

**Comment by richardwerkman** [comment]
Thank you for your PR! I'm missing the update of the [readme](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/README.md). 

It still says `--project`

File: src/Stryker.CLI/Stryker.CLI/CLIOptions.cs (hunk 1 above)
Comment (simondel) [reply src/Stryker.CLI/Stryker.CLI/CLIOptions.cs]: An example of a similar change can be found in #132. You could use that as a reference.

**Comment by Bigtalljosh** [comment]
> Thank you for your PR! I'm missing the update of the [readme](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/README.md).
> 
> It still says `--project`
😓 My mistake! Resolved now!

**Pull request merged**